department-of-veterans-affairs/caseflow
# LegacyAppeal Dismissal Due to FNOD

Completes #12211 

### Description

Adds the ability to do a Death Dismissal action on Legacy Appeals when the VACOLs SFNOD is set to a date.

### Acceptance Criteria
- [ ] Colocated Admins can do the action for any Legacy Appeal with an SFNOD assigned to the colocated team

### Testing Plan
1. Mark a LegacyAppeal with an open Colocated task with a SFNOD on the console
    `appeal.notice_of_death_date = 5.days.ago`
2. Sign in as a Colocated Admin & navigate to that appeal
3. Confirm you have the option `Death Dismissal` in the drop down
4. Confirm the action pops up a confirmation modal
5. Submitting the death dismissal results in a success message
6. On the console, confirm the appeal location is now 66 & the appeal tasks are all cancelled.
